credentials-list-directive.js

```diff
@@ -16,7 +16,9 @@ function factory($location, brAlertService, brCredentialService, config) {
     restrict: 'E',
     scope: {
       credentialType: '@brCredentialType',
-      identity: '=brIdentity'
+      identity: '=brIdentity',
```

Comment:
While we're here ... can `=brIdentity` become `<brIdentity`? Will that cause any problems?